pyslim/slim_tree_sequence.py

```diff
@@ -490,6 +490,139 @@ def recapitate(self,
             recap = tables.tree_sequence()
         return SlimTreeSequence(recap, reference_sequence=self.reference_sequence)
 
+    def continue_simulation(self, slim_time, 
+                            Ne=None, 
```

Comment:
I think you can leave out `Ne`, as we don't do anything with it, so it'll be carried along with **kwargs?

Reply:
Definitely agree with this.